modules/phoenix/src/demo/ts/ephox/phoenix/demo/ReuseDemo.ts

```diff
@@ -14,7 +14,9 @@ DomEvent.bind(SugarElement.fromDom(document), 'keydown', function (event) {
   if (event.raw.keyCode === 13) {
     Optional.from(window.getSelection()).each((selection) => {
       if (selection.rangeCount > 0) {
-        const spans = DomWrapping.reuse(SugarElement.fromDom(selection.anchorNode), selection.anchorOffset, SugarElement.fromDom(selection.focusNode), selection.focusOffset, function (elem) {
+        const anchorNode = SugarElement.fromDom(selection.anchorNode as Node);
```

Comment:
Instead of casting `as Node` could we use the `!` operator? It functions the same as a cast but it will only ever cast `T | null | undefined` to `T`. Just makes it a little clearer when reading that we're only using the cast to avoid a null check instead of any of the other things a cast could be doing